### DISCUSSION TITLE
Disable certain menu options

### DISCUSSION THREAD

**Discussion by jl-gogovapps** [outline/rich-markdown-editor#311] [General]
Is it possible to remove or disable certain menu options?  For example, the Message widgets and the Highlight don't seem to be supported by our rendering engine (flutter_markdown).

Thanks!

**Comment by jl-gogovapps** [comment]
Sorry - I see this was answered https://github.com/outline/rich-markdown-editor/discussions/305

**Comment by tommoor** [comment]
Yea, if someone was to put in a neat PR I’d definitely consider adding the
functionality back to customize menus.

But see the “important note” at the top of the Readme 😀
